src/App/App.test.js

```diff
@@ -18,7 +18,7 @@ describe('App', () => {
 
   it('should hide some header elements in default state', () => {
     const {getByText} = render(<App/>)
-    expect(getByText("Your Ratings").className).toBe("hidden");
+    expect(getByText("My Ratings").className).toBe("hidden");
```

Comment:
I think this should change back to 'Your raitings' now that the site greets the user. English nerds but it should stay in the same perspective.